content/projects/kotlin/project-10/unit-testing/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+_db_id: 490
+available_flavours:
+- kotlin
+content_type: project
+prerequisites:
+  hard: []
```

Comment:
are we sure that there are no pre-requisites? Shouldn't they have done a bit of kotlin and android work?

Reply:
Yes. Unit testing stands on its own.

Reply:
They basically do it when they are comfortable with the basics.

Reply:
Hmm. If I go visit the tutorial though it assumes that I know a whole lot of stuff. Could someone do this project as their first card on their board? If that is the case then we probably have some missing cards - it looks like people would at least need to know a bit of kotlin syntax.

Students get to choose what they want to work on next. So long as a card is not blocked by an incomplete pre-requisite then they should be able to do the work. It really looks like some knowledge is needed here. 

If we were to ask someone like Sam to learn unit testing in kotlin, what cards would she NEED to complete before she would be ready to tackle this one?

Reply:
I would probably leave it to the end. With all unit testing, the syntax is assumed knowledge. To go over exactly which topic is needed to be done is not the foundation for unit testing. I feel its better to have it on its own, then when the workshop is done, it covers all the concepts that are needed. If I was to force prerequisites, there will be assumed knowledge that will be left out.

Reply:
If you think its a must then ill take time and go through the content in detail and add them. It's not a problem.